addon/mixins/transition-mixin.js

```diff
@@ -26,203 +75,164 @@ export default Mixin.create({
     return this.get('transitionClasses').join(' ');
   }),
 
-  addClass(className, $element) {
+  addClass(className, element) {
     if (!this.get('isDestroying')) {
-      run(() => {
-        this.get('transitionClasses').addObject(className);
-      });
+      this.get('transitionClasses').addObject(className);
```

Comment:
@miguelcobain I think this `run` is required?? I was getting errors related to the run loop in testing.  Thoughts?